source/logHandler.py

```diff
@@ -266,8 +266,13 @@ def emit(self, record):
 class FileHandler(logging.FileHandler):
 
 	def handle(self,record):
-		# Only play the error sound if this is a test version.
-		shouldPlayErrorSound =  buildVersion.isTestVersion
+		import config
+		# Only play the error sound if this is a test version or if the config states it explicitely.
+		shouldPlayErrorSound = (
+			buildVersion.isTestVersion
+			# Play error sound: 1 = Yes
+			or config.conf["featureFlag"]["playErrorSound"] == 1

```

Comment:
You need to protect against `config.conf` being `None` here. This can happen if anything is logged at the error level between `logHandler` being initialized and the call to `config.initialize` in `core.main`.

Reply:
Thank you. Fixed in 9a1cb16.
I'll update the initial description regarding the inability to have the error sound played in the stable release before the log is initialized.